### PR TITLE
[Feature] Small RDXSensorSimulator Optimization

### PR DESCRIPTION
**Pull request by TomaszTB**
After learning a bit more about OpenGL, I realized we weren't actually getting anything useful from rendering depth and color separately. 
This PR changes the RDXSensorSimulator such that it only renders once, for both depth and color.

### CODE REVIEW

File: ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/simulation/sensors/RDXSensorSimulator.java [hunk 1]
```diff
@@ -173,19 +175,9 @@ public void render(RigidBodyTransform cameraTransformToWorld)
...
       }
 
-      if (depthImage != null) // If depth is enabled
+      if (depthImage != null)
       {
-         // Render and "grab" the color image
-         simulationScene.preRenderDepth(camera);
-         GL41.glViewport(0, 0, imageWidth, imageHeight);
-         simulationScene.renderDepth(RDXSceneLevel.GROUND_TRUTH.SINGLETON_SET);
-         simulationScene.postRenderDepth();
```
Comment (TomaszTB) [inline ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/simulation/sensors/RDXSensorSimulator.java:182]: The color render generates the depth data we are using. We don't have to render again for depth.

Comment (calvertdw) [reply ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/simulation/sensors/RDXSensorSimulator.java:182]: The point here is that it only renders for the ground truth scene level, so excluding virtual and modelled renderables. Virtual -> gizmos, etc.   Modelled -> point cloud, height map, etc.

Comment (TomaszTB) [reply ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/simulation/sensors/RDXSensorSimulator.java:182]: I think the color render also only renders for the ground truth scene level. Perhaps I have to double check that, but I'd be surprised if gizmos are visible in our simulated sensors.

Comment (TomaszTB) [reply ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/simulation/sensors/RDXSensorSimulator.java:182]: I took another look at the code and can confirm that the color render also only renders ground truth.

**Review by calvertdw — APPROVED** [review]

**Pull request merged**